libr/include/r_util/r_assert.h

```diff
@@ -18,14 +16,14 @@ R_API void r_assert_log(RLogLevel level, const char *fmt, ...);
 
 #define r_warn_if_reached() \
 	do { \
-		r_assert_log (R_LOG_WARNING, "(%s:%d):%s%s code should not be reached\n", \
+		R_LOG_WARN ("(%s:%d):%s%s code should not be reached\n", \
```

Comment:
i would prefer to have no macros for this.

`r_log (R_LOG_WARN, "....");`

Reply:
Personally, I'm not against these kind of macros (or inline functions, if you prefer)... they are very useful and used in other projects as well.